packages/react-log-viewer/src/LogViewer/utils/utils.tsx

```diff
@@ -41,3 +41,32 @@ export const parseConsoleOutput = (data: string) => {
 };
 
 export const escapeString = (inputString: string): string => inputString.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&'); // eslint-disable-line
+
+/* eslint-disable no-control-regex */
+export const ansiRegex = new RegExp(/[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-PRZcf-nqry=><]/g);
+
+export const isAnsi = (inputString: string) => inputString.match(ansiRegex);
+
+export const stripAnsi = (inputString: string): string => inputString.replace(ansiRegex, '');
+
+export const splitAnsi = (inputString: string): string[] =>
+  inputString.split(new RegExp(/([\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-PRZcf-nqry=><])/g));
```

Comment:
One more little refactor :) you can pull out the regex string, also you're mixing the regex literal `//` with `RegExp function, which still works but usually it's either literal or RegExp with string.
```
const ansiRegExp = `[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-PRZcf-nqry=><]`;
export const splitAnsi = (inputString: string): string[] =>
  inputString.split(new RegExp(`(${ansiRegExp})`, 'g'));
```

Reply:
Updated, thank you!